services/galley/src/Galley/API/LegalHold.hs

```diff
@@ -54,12 +54,12 @@ createSettingsH (zusr ::: tid ::: req ::: _) = do
 createSettings :: UserId -> TeamId -> NewLegalHoldService -> Galley ViewLegalHoldService
 createSettings zusr tid newService = do
   assertLegalHoldEnabled tid
-  membs <- Data.teamMembers tid
-  let zothers = map (view userId) membs
-  Log.debug $
-    Log.field "targets" (toByteString . show $ toByteString <$> zothers)
-      . Log.field "action" (Log.val "LegalHold.createSettings")
-  void $ permissionCheck zusr ChangeLegalHoldTeamSettings membs
+  zusrMembership <- Data.teamMember tid zusr
+  -- let zothers = map (view userId) membs
+  -- Log.debug $
+  --   Log.field "targets" (toByteString . show $ toByteString <$> zothers)
+  --     . Log.field "action" (Log.val "LegalHold.createSettings")
```

Comment:
@fisx I am not sure what we should be doing here. But there are a few places I have commented the logging code out. Any ideas? /cc @tiago-loureiro

Reply:
I was wondering myself, and decided that leaving the comment is the best course of action (if you don't want it to do any actual debug logging).  It's not very distracting, and a good suggestion for people coming here when debugging.